### ISSUE TITLE
Notification is broken

### ISSUE THREAD

**Issue by abixalmon** [flarum/core#1567]
latest dev-master build.

```
GET /api/notifications

<b>Warning</b>:  array_key_exists(): The first argument should be either a string or an integer in <b>/flarum/vendor/illuminate/support/Arr.php</b> on line <b>151</b>

<b>Warning</b>:  strpos() expects parameter 1 to be string, array given in <b>/flarum/vendor/illuminate/support/Arr.php</b> on line <b>292</b>

<b>Warning</b>:  explode() expects parameter 2 to be string, array given in <b>/flarum/vendor/illuminate/support/Arr.php</b> on line <b>296</b>

<b>Warning</b>:  Invalid argument supplied for foreach() in <b>/flarum/vendor/illuminate/support/Arr.php</b> on line <b>296</b>

<b>Fatal error</b>:  Uncaught Error: Class name must be a valid object or a string in /flarum/vendor/illuminate/database/Eloquent/Concerns/HasRelationships.php:653
Stack trace:
#0 /flarum/vendor/illuminate/database/Eloquent/Concerns/HasRelationships.php(224): Illuminate\Database\Eloquent\Model-&gt;newRelatedInstance(Array)
#1 /flarum/vendor/illuminate/database/Eloquent/Concerns/HasRelationships.php(194): Illuminate\Database\Eloquent\Model-&gt;morphInstanceTo(Array, 'subject', 'subjectModel', 'subject_id')
#2 /flarum/vendor/flarum/core/src/Notification/Notification.php(137): Illuminate\Database\Eloquent\Model-&gt;morphTo('subject', 'subjectModel', 'subject_id')
#3 /flarum/vendor/illuminate/database/Eloquent/Builder.php(546): Flarum\Notification\Notification-&gt;subject()
#4 [internal function]: Illuminate\Database\Eloquent\Builder-&gt;Illuminate\Database\Eloquent\{closure}()
#5 /flarum/vendor/illuminate/database/Eloquent/Relati in <b>/flarum/vendor/illuminate/database/Eloquent/Concerns/HasRelationships.php</b> on line <b>653</b>
```

**Comment by luceos** [comment]
Please post all information requested in the bug report template! For instance knowing your php version would make a world of a difference here...

**Comment by tobscure** [comment]
Fixed in 986d811a1615a2715b89c463ee3ee69e3440ac50
